cephalopod/factory.py

```diff
@@ -19,7 +19,24 @@
 def make_app():
     """Returns a :class:`CustomFlask` application instance that is properly configured."""
     app = Flask('cephalopod')
-    app.config.from_pyfile('defaults.cfg')
+    # Defaults
+    app.config.update({
+        'BABEL_DEFAULT_TIMEZONE': 'UTC',
+        'BABEL_DEFAULT_LOCALE': 'en_GB',
+        'APP_NAME': 'Cephalopod',
+        'ASSETS_DEBUG': False,
+        'DEBUG': False,
```

Comment:
this one can be omitted, that's already the default. forgot to remove that in defaults.cfg